manifests/init.pp

```diff
@@ -91,6 +91,9 @@
 #   this parameter with your SSL key's location before deploying this server in a production 
 #   environment.
 #
+# @param default_ssl_reload_on_change
+#   Enable reloading of apache if the content of ssl files have changed.
```

Comment:
I think this should clarify this **only** sets the default value for vhosts. If you don't have any SSL vhosts or use `mod_ssl` to configure the SSL certificates, it does nothing.